docs/API.md

```diff
@@ -1093,102 +1263,93 @@ its address.*
+rather change the whole set of options through `SRTO_TRANSTYPE` option.
 
-- **[SET]** - The type of congestion controller used for the transmission for
-that socket. Its type must be exactly the same on both connecting parties,
-otherwise the connection is rejected. 
-- ***TODO: might be reasonable to allow an "adaptive" congestion controller,
-which will make the side that sets it accept whatever controller type is set
-by the peer, including different per connection***
 
 ---
 
-| OptName               | Since | Binding | Type  | Units  | Default          | Range  |
-| --------------------- | ----- | ------- | ----- | ------ | ---------------- | ------ |
-| `SRTO_SNDBUF`         |       | pre     | `int` | bytes  | 8192 × (1500-28) |        |
+| OptName           | Since | Binding | Type       |  Units  |  Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | ---------- | ------- | --------- | ------ | --- | ------ |
+| `SRTO_SNDBUF`     |       | pre     | `int32_t`  | bytes   |8192 bufs  | *      | RW  | GSD+   |
 
-- Send Buffer Size.  ***Warning: configured in bytes, converted in packets, when 
-set, based on MSS value. For desired result, configure MSS first.*** 
+- Sender Buffer Size. See `SRTO_RCVBUF` for more information.
 
 ---
 
-| OptName           | Since | Binding | Type      | Units  | Default  | Range  |
-| ----------------- | ----- | ------- | --------- | ------ | -------- | ------ |
-| `SRTO_SNDDATA`    |       | n/a     | `int32_t` | pkts   | n/a      | n/a    |
+| OptName           | Since | Binding | Type       |  Units  |  Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | ---------- | ------- | --------- | ------ | --- | ------ |
+| `SRTO_SNDDATA`    |       |         | `int32_t`  | pkts    |           |        | R   | S      |
 
-- **[GET]** - Size of the unacknowledged data in send buffer.
+- Size of the unacknowledged data in send buffer.
 
 ---
 
-| OptName               | Since | Binding | Type  | Units  | Default  | Range  |
-| --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
-| `SRTO_SNDDROPDELAY`   | 1.3.2 | pre     | `int` | ms     | 0        | -1..   |
+| OptName              | Since | Binding | Type       |  Units  |  Default  | Range  | Dir | Entity |
+| -------------------- | ----- | ------- | ---------- | ------- | --------- | ------ | --- | ------ |
+| `SRTO_SNDDROPDELAY`  | 1.3.2 | pre     | `int32_t`  | ms      | *         | -1..   | W   | GSD+   |
 
-- **NB:** The default [live mode](#transmission-method-live) settings set `SRTO_SNDDROPDELAY` to 0.
-The [buffer mode](#transmission-method-buffer) settings set `SRTO_SNDDROPDELAY` to -1.
+- Sets an extra delay before TLPKTDROP is triggered on the data sender.
+Extra, that is, it is added to the default drop delay time interval value.
+The longer the time, the more probable that packets would be retransmitted
+uselessly because they will be dropped by the receiver anyway.
```

Comment:
```suggestion
- Sets an extra delay before TLPKTDROP is triggered on the data sender.
This delay is added to the default drop delay time interval value. Keep in mind 
that the longer the delay, the more probable it becomes that packets would be 
retransmitted uselessly because they will be dropped by the receiver anyway.
```